corehq/apps/repeaters/models.py

```diff
@@ -324,6 +261,13 @@ class FormRepeater(Repeater):
 
     """
 
+    class Formats(object):
+        formats = {
+            'form_xml': (FormRepeaterXMLPayloadGenerator, _('XML')),
+            'form_json': (FormRepeaterJsonPayloadGenerator, _('JSON')),
+        }
+        default_format = 'form_xml'
```

Comment:
@NoahCarnahan So I tried going with that first, but it turns out that couchdbkit.Document does not let you have top-level, non-private (i.e. non-`_`-prefixed) properties that are dicts or lists. That's because if it's a type that it naively detects as JSON-serializable (ints, strings, lists, dicts, etc.) then it treats it as a *Property with a constant value (with the effect that it saves these to the db). This is behavior I inherited from couchdbkit when I rewrote its schema library to be jsonobject-based, and I intentionally kept it for backwards compatibility (we do rely on that behavior here and there).

Anyway, so as a result, I'm sticking it inside this class. Open to better ideas!